generator/ServiceClientGeneratorLib/Generators/Marshallers/RestXmlResponseUnmarshaller.tt

```diff
@@ -172,6 +172,10 @@ namespace <#=this.Config.Namespace #>.Model.Internal.MarshallTransformations
 #>
 					if (context.TestExpression("<#=DetermineXmlMarshallName(member)#>", targetDepth))
 					{
+                        if (response.<#=member.PropertyName#> == null)
```

Comment:
I don't understand why are we initializing a value for these collections. The goal in V4 is to not initialize collections to avoid unused allocations and communicate to users what actually had a value from the service.